src/NServiceBus.Core/App_Packages/Particular.Licensing/FindActiveLicense/ActiveLicense.cs

```diff
@@ -20,15 +20,22 @@ public static ActiveLicenseFindResult Find(string applicationName, params Licens
             var licenseSourceResultToUse = LicenseSourceResult.DetermineBestLicenseSourceResult(results.ToArray());
             if (licenseSourceResultToUse != null)
             {
-                activeLicense.Report.Add($"Selected active license from {licenseSourceResultToUse.Location}");
+                var selectedLicenseReportItem = $"Selected active license from {licenseSourceResultToUse.Location}";
+                activeLicense.Report.Add(selectedLicenseReportItem);
```

Comment:
imho we need to introduce a breaking change at some time if we ever want to avoid the duplicated entries (which I think we should do). An alternative would be to introduce two new properties which split the log entries between the debug and the info logs and the existing one which is those two combined.

I know this is nitpicking, but now that we have spent so much time and effort on coming up with proper ways to optimize the logging, it feels weird to live with such a compromise :D